.travis.yml

```diff
@@ -3,12 +3,17 @@ language: python
 python:
   - "3.6"
   - "3.5"
-  - "3.4"
-  - "3.3"
   - "2.7"
-  - "2.6"
```

Comment:
?!

Reply:
Same for this - rebase cleaned this right up.